x-pack/plugins/notifications/server/services/email_service.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface IEmailService {
+  sendPlainTextEmail(payload: PlainTextEmail): Promise<void>;
+}
+
+export interface PlainTextEmail extends Record<string, unknown> {
```

Comment:
Why is that extending `Record<string, unknown>`?

Reply:
It doesn't make much sense, and I don't remember why I added it on the first place.
I will remove it, thanks for spotting it!